microsoft-edge/devtools-guide-chromium/evaluate-performance/selector-stats.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Analyze selector performance during Recalculate Style events
+description: Use the Selector Stats feature in the Performance tool to understand which of your CSS selectors are taking up the most time during Recalculate Style events.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 01/09/2023
+---
+# Analyze selector performance during Recalculate Style events
+
+The **Performance** tool highlights long tasks with red triangles to indicate work on the main thread that takes too long and has slow performance.  
+
+![Long Task indicators for events on the main thread with slow performance.](../media/evaluate-performance-performance-animation-frame-fired.msft.png)
```

Comment:
Can red boxes be added?  No idea what to look at in this capture.  Where am I supposed to be looking?

GH rendered:
https://github.com/MicrosoftDocs/edge-developer/blob/user/zoghadya/selector-stats/microsoft-edge/devtools-guide-chromium/evaluate-performance/selector-stats.md#analyze-selector-performance-during-recalculate-style-events

Reply:
I would also suggest creating a new image instead of reusing an existing one.
Our current way of dealing with images is that each md file should really have its own image folder with its own images. 
Reusing images across articles usually leads to problems down the line, even if it feels like we're saving time and cutting on file numbers initially.

Reply:
caution, Outdated comment thread.
"its own image folder", that would be:
microsoft-edge/devtools-guide-chromium/evaluate-performance/selector-stats-images/

Reply:
@captainbrosset - I entered a fresh non-Outdated [comment](https://github.com/MicrosoftDocs/edge-developer/pull/2399/files#r1068561465) to create a dedicated images dir.

Reply:
> I would also suggest creating a new image instead of reusing an existing one. Our current way of dealing with images is that each md file should really have its own image folder with its own images. Reusing images across articles usually leads to problems down the line, even if it feels like we're saving time and cutting on file numbers initially.

fixed